inst/NEWS.Rd

```diff
@@ -3,8 +3,14 @@
 \newcommand{\cpkg}{\href{http://CRAN.R-project.org/package=#1}{\pkg{#1}}}
 \section{Changes in version 2.8-4 (GitHub-only-release date:?-?-2022)}{
   \itemize{
-    \item Part 5: dayborder != 0 is now handled in g.part5.wakesleepwindow
-    \item Part 5: excludefirstlast.part5 is now applied per window type in g.report.part5
+    \item Minor fix to g.getbout because it included one epoch beyond the period of interest in the identified bout.
+    \item Part 2: Added functionality to impute invalid time segments by zero.
+    \item Part 2: Added functionality to exclude non-wear time from part 2 reports.
+    \item Part 2: Added functionality derive inactivity and light PA bouts in part 2 when the devices are not worn during sleep (no need to define the SPT in parts 3 & 4).
+    \item Revised bout detection in bout.metric = 6
```

Comment:
This overlaps with "Minor fix to g.getbout" above, do we need to merge these two items?

Reply:
Removed the bout fixes from this branch as they were already specified in [PR #716](https://github.com/wadpac/GGIR/pull/716)